src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSTracingPipelineHandler.cs

```diff
@@ -172,6 +177,82 @@ private static void AddRequestSpecificInformation(Activity activity, IRequestCon
             SnsRequestContextHelper.AddAttributes(
                 requestContext, AWSMessagingUtils.InjectIntoDictionary(new PropagationContext(activity.Context, Baggage.Current)));
         }
+        else if (AWSServiceType.IsBedrockRuntimeService(service))
+        {
+            activity.SetTag(AWSSemanticConventions.AttributeGenAiSystem, "aws_bedrock");
+        }
+        else if (AWSServiceType.IsBedrockAgentService(service))
```

Comment:
To my understanding, this is not the right place extracting bedrockAgent Operation related attributes. Could you please address the comment as above?